templates/solutions/ai/index.html

```diff
@@ -321,22 +321,22 @@ <h2>Run on any cloud</h2>
                 }}
               </div>
               <div class="p-logo-section__item">
-                {{ image(url="https://assets.ubuntu.com/v1/dc51883b-microsoft-azure-2021.png",
-                                alt="Microsoft Azure",
-                                width="95",
-                                height="313",
+                {{ image(url="https://assets.ubuntu.com/v1/210f44e4-microsoft-azure-new-logo.png",
+                                alt="Azure",
+                                width="252",
+                                height="288",
                                 hi_def=True,
-                                loading="lazy",
+                                loading="auto|lazy",
```

Comment:
```suggestion
                                loading="lazy",
```
Images not immediately visible in the viewport should be lazy